### PR TITLE
nolo tlc ciel d9 9_2b

### PR DESCRIPTION
**Pull request by nolo16**
the deeper you go, the worse this file gets

### CODE REVIEW

File: script/Ciel/Day 9/09_01_CIEL09_2B.txt [hunk 1]
```diff
@@ -1204,17 +1207,17 @@ She's saying that even though she's doesn't know that Arcueid is a good person.
...
 -- Page 238, Offset 22749.
 -- だって、アルクェイドはいいヤツだし！　俺を殺そうとするとか、そんなコト絶対に、」
-Arcueid is a good person, you know! She wouldn't never try to kill me or―――"
+Arcueid is a good person, you know! She would never try to kill me or―――"
 }
 [sha:9556227980f802a9a775fe4680975e167ebc8499]{
 -- Page 239, Offset 22750.
 -- 「ですから、その彼女自身が危険なんです！　いいかげん目を覚ましてください！
-"That's why I said that her very being was dangerous! Enough already, please be lucid!
+"I told you, didn't I?! Arcueid herself is dangerous! Enough already, please open your eyes!
```
Comment (Avuxo) [inline script/Ciel/Day 9/09_01_CIEL09_2B.txt:1215]: either drop or italicize "please".

File: script/Ciel/Day 9/09_01_CIEL09_2B.txt [hunk 2]
```diff
@@ -1289,12 +1292,12 @@ What exactly?
 [sha:06651dd71a2f5388b7571fd298f129137827e002]{
 -- Page 244, Offset 22766.
 -- たとえ彼女が危険ではないとしても、遠野くんが戦うこと自体が危険じゃないですか」
-Even supposing that she isn't a danger, the very fact that you engage in battle is dangerous, don't you think?"
+Even supposing that she herself isn't dangerous, the very fact that you're engaging in battle is, don't you think?"
```
Comment (Avuxo) [inline script/Ciel/Day 9/09_01_CIEL09_2B.txt:1295]: italicize that "is" at the end

File: script/Ciel/Day 9/09_01_CIEL09_2B.txt [hunk 3]
```diff
@@ -1369,12 +1372,12 @@ I stand up, yelling at Senpai.
...
 -- Page 250, Offset 22782.
 -- 「感謝？　それこそあり得ません。あんな化け物に感謝してどうするんですか。そもそも元凶は彼女です、あれぐらい当然の義務でしょう」
-"Thankful? If there's one thing I won't do, this is it. WHy apologize to such a monster? She's the culprit behind all of this anyway, just as well that she took care of something so trivial as her obvious duty!"
+"Gratitude? Absolutely not. What good will apologizing to that monster do, anyway? This is all her fault anyway, so it's only natural that she should clean up her own mess."
 }
 [sha:28e2903cada5a8f7d7b62a22a5b9b48303be45c0]{
 -- Page 251, Offset 22783.
 -- 「ああ、そういうコトですか。なるほど、納得しました。どうしてアルクェイドが街を守ったのか。あれは貴方を騙すための布石だったんですね。ああやって“人間らしい”フリをすれば、遠野くんがころっと騙されるから」
-"Ahh, is that it? I see now. I understand why Arcueid protected the city. It was the first step in deceiving you. By doing something like this and pretending to act like a human, you got fooled hook, line and sinker, Tohno-kun."
+"Ahh, is that it? I see now. I understand why Arcueid protected the city. It was the first step in deceiving you, Tohno-kun. By doing that and pretending to act like a human, she's got you fooled hook, line, and sinker."
```
Comment (Avuxo) [inline script/Ciel/Day 9/09_01_CIEL09_2B.txt:1380]: fooled, hook, line, and sinker.

File: script/Ciel/Day 9/09_01_CIEL09_2B.txt [hunk 4]
```diff
@@ -1384,12 +1387,12 @@ I stand up, yelling at Senpai.
 [sha:fd947796199076bddf3f5ab66b0c5e9a504baef8]{
 -- Page 252, Offset 22785.
 -- 今のが俺への皮肉なのか、本気で言ったものなのかはどうでもいい。
-I don't give a damn whether she said it seriously or ironically.
+I don't give a damn whether she was being serious or ironic.
```
Comment (Avuxo) [inline script/Ciel/Day 9/09_01_CIEL09_2B.txt:1390]: i don't think 皮肉 is "irony", i think it's like "sarcastic bad faith", which i think is supported by the 俺へ preceding it.
maybe something like
> I don't give a damn whether she was being serious or just trying to rile me up.

File: script/Ciel/Day 9/09_01_CIEL09_2B.txt [hunk 5]
```diff
@@ -1436,27 +1439,27 @@ Huh, that's a line I heard somewhere else.
...
 -- Page 258, Offset 22797.
 -- 「ああそうですかっ！　わたしも同じぐらい不愉快です！
-"Ahh really! I'm revolted the same!
+"Ahh, really! Well, the same goes for me!
 }
 [sha:15af17188dced534624874f5921fd09118653a9d]{
 -- Page 258, Offset 22798.
 -- そんなに言うならアルクェイドに騙されて、いいように使われたらどうですか!?　そんな人を心配するのもバカらしいですから、わたし！」
-If you want it that much, then about you go to Arcueid, go get deceived by her, go be used by her however she wants, why don't you!? I'm the one who was stupid enough to care about someone like you!"
+If you want it that much, then how about you go and let yourself be deceived and used by Arcueid?! I was stupid to be worried about someone like you!"
```
Comment (Avuxo) [inline script/Ciel/Day 9/09_01_CIEL09_2B.txt:1457]: Arcueid!

File: script/Ciel/Day 9/09_01_CIEL09_2B.txt [hunk 6]
```diff
@@ -1466,32 +1469,32 @@ If you want it that much, then about you go to Arcueid, go get deceived by her,
...
 -- Page 260, Offset 22801. Glued.
 -- 遠野くんなんか絶交です！
-I don't want to see you again, Tohno-kun!
+I never want to see you again, Tohno-kun!
 }
 [sha:d5542aeca7a65179d917407d241866b1554883bc]{
 -- Page 260, Offset 22802.
 -- ええ、こちらも清々します！　遠野くんなんて頼りにならないし、弱いし、放っておけないコトばっかりするし！　二度と話しかけたり様子を見に行ったり助けたりしませんから、そのつもりで！」
-I'll feel refreshed too, for sure! At any rate, you're unreliable and you're weak and you're always doing stuff where I can't leave you to it, Tohno-kun! I don't have the intention to ever talk to you again, I won't check up on you and I won't help you anymore!"
+I'm relieved too! You're unreliable, and you're weak, and I always have to keep an eye out for you anyway! I'm never talking to you again, never checking up on you again, and never helping you again!"
```
Comment (Avuxo) [inline script/Ciel/Day 9/09_01_CIEL09_2B.txt:1477]: Yeah, that's a weight off my chest, too!

File: script/Ciel/Day 9/09_01_CIEL09_2B.txt [hunk 7]
```diff
@@ -1562,22 +1565,22 @@ Oh, I know. You're holding a bag from the school store, there wasn't any good re
...
 -- Page 270, Offset 22822.
 -- 昼休み終了まであと10分。
-There's still ten more minutes before the end of lunch break.
+Ten minutes to go before the end of lunch break.
 }
 [sha:bd7b67b79045e63e5bba8017643f130d4a6a1b07]{
 -- Page 270, Offset 22823.
 -- 精神的にも身体的にも、とても食べられる状況じゃない。
-I am not in a state to be eating, both emotionally and physically speaking.
+I'm not really in the condition to eat, both emotionally and physically speaking.
```
Comment (Avuxo) [inline script/Ciel/Day 9/09_01_CIEL09_2B.txt:1583]: I'm not really in any condition to eatーーphysically, or emotionally.

**Review by Avuxo — APPROVED** [review]
大tsukare, on god

File: script/Ciel/Day 9/09_01_CIEL09_2B.txt [hunk 8]
```diff
@@ -37,37 +37,37 @@ You are misunderstanding her nature. Aren't you thinking that because she doesn'
...
 -- Page 153, Offset 22521.
 -- いいでしょう。部外者である遠野くんに説明する事は避けていましたが、今回は特例です。
-Listen, I tried to avoid explaining things to an outsider like you, but I'll make an exception this time.
+Listen up. I tried to avoid explaining things because you're an outsider, but I'll make an exception this time.
 }
 [sha:ea06c80d0ead5dd1edc6efbfebc509b9a53197b9]{
 -- Page 153, Offset 22522.
 -- 貴方がこの先も吸血鬼に関わるというのなら、彼女と彼女の敵―――ロアについてお話しします」
-If you continue to involve yourself with vampires after this, I'll need to talk about her and her enemy, Roa."
+If you still intend on involving yourself with vampires from here on, then I'll have to tell you about Arcueid and her enemy, Roa."
```
[the remark this answers is not in the document]
Comment (nolo16) [reply script/Ciel/Day 9/09_01_CIEL09_2B.txt:70]: dropped the have to part

File: script/Ciel/Day 9/09_01_CIEL09_2B.txt [hunk 9]
```diff
@@ -285,32 +287,32 @@ And once the host body reaches the point where it possesses sufficient intellige
...
 -- Page 174, Offset 22566.
 -- 「基本的にはそうです。移植は転生先の肉体が胎児の時に行われます。それは医学的な工程ではなく、自然のサイクルに近い工程です。
-"More or less. The transplant, less a medical process than a natural cycle, occurs when the host is a fetus.
+"More or less. It occurs while the host is a fetus, and is more like a natural cycle than a medical process.
 }
 [sha:e50aae64c07fb70cf8aabc0b2ab73dc97fa8bff4]{
 -- Page 174, Offset 22567.
 -- さきほど全情報と言いましたが、ようは魂という名の電波と思ってください。電波を発信するのも受信するのも人間の脳」
-I said that he transfers all of his data, but basically, think of it as <waves| s o u l>, with the human brain both transmitter and receiver."
+I said that he transfers all of his data, but think of it like this: he transfers his soul as waves, and the human brain is the receiver and the transmitter."
```
[the remark this answers is not in the document]
Comment (nolo16) [reply script/Ciel/Day 9/09_01_CIEL09_2B.txt:300]: i will try ™️ this but i think its gonna cause overflow

File: script/Ciel/Day 9/09_01_CIEL09_2B.txt [hunk 10]
```diff
@@ -285,32 +287,32 @@ And once the host body reaches the point where it possesses sufficient intellige
...
 }
 [sha:a46528804e1b3564df4972da293d574b7910cb5e]{
 -- Page 174, Offset 22568.
 -- 「ロアはその肉体が死滅した際、魂を拡散させて次の肉体に転移します。転生先が決まっていればその肉体に。決まっていなければ一定条件を満たした胎児にランダムに。
-"When Roa's body dies, he diffuses his soul, and transfers to his next body, either one chosen beforehand, or otherwise, to a random fetus that fulfills some conditions.
+"When Roa dies, his soul diffuses and transfers to his next body. If he has already determined a new host, then he sends it to that body, if not, he chooses a random fetus that partly satisfies his conditions.
```
[the remark this answers is not in the document]
Comment (nolo16) [reply script/Ciel/Day 9/09_01_CIEL09_2B.txt:305]: i assumed the original "partly" came from later on where she says the other condition isnt an absolute, but i guess its kinda unnecessary

File: script/Ciel/Day 9/09_01_CIEL09_2B.txt [hunk 11]
```diff
@@ -784,7 +786,7 @@ Her emotionless eyes announced what she was going to say next.
 [sha:c8728165d3267fb3687fd3d50b982ff487c2caa2]{
 -- Page 211, Offset 22664.
 -- 彼女は八百年前に一度だけ過ちを犯した。真祖の王族でありながら人間の司祭だったロアの甘言に乗り、杯に汲まれたロアの血を飲んでしまった」
-Eight hundred years ago, she made one single mistake. The True Ancestor of royal blood, enraptured by Roa's sweet words, drank from the chalice filled with the blood of the human priest."
+Eight hundred years ago, she made a single mistake. The True Ancestor of royal blood, enraptured by Roa's sweet words, drank from the chalice filled with the blood of the human priest."
```
[the remark this answers is not in the document]
Comment (nolo16) [reply script/Ciel/Day 9/09_01_CIEL09_2B.txt:789]: i read this and was like how the FUCK does that work

File: script/Ciel/Day 9/09_01_CIEL09_2B.txt [hunk 12]
```diff
@@ -820,42 +822,42 @@ That vampires aren't that big of a deal, no matter their undying bodies or immor
...
 -- 真祖とは精霊そのもの。地上で起きるあらゆる自然現象の化身。この<惑|ほ><星|し>の分身と言ってもいい」
 // Quintuplets TA when?
-True Ancestors are the quintessential elementals, the personification of the various natural phenomenons that occur on the surface. You could call them incarnations of the planet."
+True Ancestors are the quintessential elementals, the personification of the various natural phenomena that occur on the surface. You could call them incarnations of the planet."
 }
 [sha:1558ca8823848272bb47ccd87fef3e77d30adb49]{
 -- Page 215, Offset 22672.
 -- 「わたしたち人間は自然との共存から独立、自立する事で繁栄してきました。星の恩恵を受けながらも資源を略奪し、星が滅びようとわたしたちは滅びない未来を手に入れた。
-"We, humans, have prospered from becoming independent, separated from our common existence with nature. Receiving the blessings of the planet while plundering its resources, we have taken into our hands a future where we won't be extinguished even if the planet perishes.
+"We, humans, have prospered due to our independence and separation from our coexistence with nature. Receiving the blessings of the planet while plundering its resources, we have attained a future where we will not be annihilated even if the planet perishes.
```
[the remark this answers is not in the document]
Comment (nolo16) [reply script/Ciel/Day 9/09_01_CIEL09_2B.txt:830]: current reads right to me but maybe "attaining self-sufficiency and independence from our coexistence with nature" would be more apt

**Pull request closed without merging**